_posts/en/newsletters/2023-02-01-newsletter.md

```diff
@@ -0,0 +1,204 @@
+    users by making the heuristic less reliable.
+
+    However, payjoin isn't as flexible as typical Bitcoin payments.
+    Most typical payments can be sent when the receiver is offline, but
+    payjoin requires the receiver to be online to contribute and sign
+    for their inputs.  The existing payjoin protocol also requires the
+    receiver to accept HTTP requests at a network address accessible to
+    the spender, which is commonly accomplished by the receiver running
+    a webserver on a public IP address which contains payjoin-compatible
+    software.  As mentioned in [Newsletter #132][news132 payjoin], one
+    suggestion for increasing the use of payjoin would be to allow
+    payjoin on a more P2P basis between typical end-user wallets.
+
+    Gould suggests building into payjoin-compatible wallets a
+    lightweight HTTP server with [noise protocol][] encryption support
+    plus the ability to use the [TURN protocol][] to traverse [NAT][].
+    This would allow two wallets to communicate interactively for the
+    brief period it takes to create a payjoin payment, with no need for
+    a long-term webserver.  This does not allow payjoins to be created
+    while the receiver is offline, although Gould does suggest
+    investigating the [nostr protocol][] for future proposals to enable
+    "async payjoin".
+
+    As of this writing, no replies to the proposal have been posted to
+    the mailing list.
+
+- **LN async proof of payment:** as mentioned in [last week's
+  newsletter][news235 async], LN developers are seeking a method for
+  sending [async payments][topic async payments] that provide the
+  spender with proof they paid the receiver.  An async payment would
+  involve a spender (Alice) sending an LN payment to a receiver (Bob)
+  through a normal series of forwarding hops---including a Lightning
+  Service Provider (LSP) that would hold the payment for Bob if he was
+  offline at the moment.  When Bob notifies the LSP that he's back
+  online, the LSP would begin forwarding the payment the rest of the way
+  to Bob.
+
+    A challenge with this approach in the current [HTLC][topic
+    htlc]-based LN is that, if Bob is offline, he can't provide Alice
+    with a payment-specific invoice that references a secret he's
+    chosen.  Alice can instead choose her own secret and include it in
+    the async payment she sends Bob---this is called a [keysend][topic
+    spontaneous payments] payment---but since Alice knew the keysend
+    secret all along, she can't use her knowledge of it as proof that
+    she paid Bob.  Alternatively, Bob could pre-generate several
+    standard invoices and give them to his LSP, who could distribute
+    them to potential spenders like Alice.  Paying those invoices would
+    generate proof of payment when Bob ultimately accepted payment, but
+    it would allow the LSP to distribute the same invoice to several
+    spenders, causing them all to pay same secret.  When the LSP learned
```

Comment:
```suggestion
    spenders, causing them all to pay the same secret.  When the LSP learned
```

Reply:
Feel free to ignore: this paragraph uses the past tense ("Alice know...Bob accepted...LSP learned"), while the next paragraph uses the present tense ("Bob comes back...Bob verifies...he accepts...she receives"), for what looks like the same kind of construction.